executor/native_linux_executor.cpp

```diff
@@ -67,7 +67,8 @@ NativeLinuxExecutor::NativeLinuxExecutor(
     const fs::path &path_to_write_input,
     u32 afl_shm_size,
     u32  bb_shm_size,
-    bool record_stdout_and_err
+    bool record_stdout_and_err,
+    fuzzuf::executor::EnvironmentVariables &&environment_variables_
```

Comment:
In my opinion, it is easier to imagine what `environment_variables_` actually is if we directly use `std::vector<std::string>` instead of the type alias `fuzzuf::executor::EnvironmentVariables`. 
This is because, when we see the original type `EnvironmentVariables`, we have to investigate what it is by searching its definition. 

Are you going to change the definition of `EnvironmentVariables` somehow later? 
Like transforming it to `std::map<std::string, std::string>` or defining a new struct?

Reply:
fixed